libbpf-rs/src/map.rs

```diff
@@ -61,6 +61,20 @@ impl OpenMap {
         unsafe { libbpf_sys::bpf_map__set_inner_map_fd(self.ptr, inner.fd()) };
     }
 
+    /**
+     * Reuse a map created within the same userspace process.
+     * To use maps across userspace processes, pin the map first and use "reuse_pinned_map".
```

Comment:
and again, it's one of the ways, and it has nothing to do with `reuse_fd` API. I'm not sure doc comment on reuse_fd is the right place to explain all possible ways to share maps between processes, tbh.